packages/metadata/networks.json

```diff
@@ -213,7 +213,7 @@
         "logsQueryRange": 20000,
         "explorer": "https://gnosisscan.io",
         "subgraphV1": {
-            "cliName": "gnosis",
+            "cliName": "xdai",
```

Comment:
I think this is a regression - it used to be xdai, was changed to gnosis.
If goldsky still requires xdai, they probably use an old version. Changing here may cause breakage with other vendors.

Reply:
Yep, needs an exception for Goldsky in the subgraph deploy script